op-challenger/game/fault/trace/cannon/executor.go

```diff
@@ -157,9 +157,9 @@ func runCmd(ctx context.Context, l log.Logger, binary string, args ...string) er
 	return cmd.Run()
 }
 
-// findStartingSnapshot finds the closest snapshot before the specified traceIndex in snapDir.
+// FindStartingSnapshot finds the closest snapshot before the specified traceIndex in snapDir.
 // If no suitable snapshot can be found it returns absolutePreState.
-func findStartingSnapshot(logger log.Logger, snapDir string, absolutePreState string, traceIndex uint64) (string, error) {
+func FindStartingSnapshot(logger log.Logger, snapDir string, absolutePreState string, traceIndex uint64) (string, error) {
```

Comment:
I see we're using this in a few packages outside `cannon`, would this be a good candidate to refactor out as well?

Reply:
Created a `utils` package. Refactored out at https://github.com/ethereum-optimism/optimism/pull/10094/commits/d536958da6ca05b2f7e2ba2e4f3b5456d78d8d7f, https://github.com/ethereum-optimism/optimism/pull/10094/commits/ffc3f1fc8b79b16d24f4f2e06dfdc4b0dc7933d3.